web/themes/custom/pineapple/sass/components/paragraphs/_image-grid.scss

```diff
@@ -0,0 +1,50 @@
+.field--name-field-p-ig-item {
+  @include breakpoint($lap) {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .field__item {
+    position: relative;
+    width: 100%;
+    margin-right: $unit / 2;
+
+    &.wide {
+      @include breakpoint($lap) {
+        width: 60%;
+      }
```

Comment:
Update width to 64.5% and remove breakpoint

Reply:
I adjusted to 64% and 32% respectively, for spacing consistency.